test/Elastic.Apm.AspNetCore.Tests/MicrosoftExtensionsConfigTests.cs

```diff
@@ -30,7 +30,7 @@ public class MicrosoftExtensionsConfigTests
 		public void ReadValidConfigsFromAppSettingsJson()
 		{
 			var config = new MicrosoftExtensionsConfig(GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_valid.json"),
-				new TestLogger());
+				new NoopLogger(), "test");
```

Comment:
I would add an `internal` constructor overload to `MicrosoftExtensionsConfig` that doesn't require `environmentName` parameter - it will be easier to maintain.

Reply:
From one point of view, it would be easier to maintain, however, from another point of view it need to be as required parameter. In case of adding internal constructor overload it will be possible to call constructor w/o environment name inside asp.net core configuration. By this reason, I prefer left only one constructor